.gitignore

```diff
@@ -8,6 +8,7 @@ node_modules
 # testing
 /coverage
 packages/*/coverage
+cypress/results
```

Comment:
There's also `cypress/videos` directory created. Might be good to add it too.

Reply:
Is there one? Isn't it from one of your old runs?

I intentionally redirected the videos to inside the `cypress/results/videos` folder so we keep everything at one place.

I guess you ran e2e before which created that folder but if it's not the case, LMK.

Reply:
Yeah, that's probably that, I'll check in the moment but feel free to merge it already

Reply:
That was it indeed :man_facepalming: